utilities/merge_operators/cassandra/merge_operator.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2017-present, Facebook, Inc.  All rights reserved.
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree. An additional grant
+// of patent rights can be found in the PATENTS file in the same directory.
+
+#pragma once
+#include "rocksdb/merge_operator.h"
```

Comment:
Wondering if file name collisions (merge_operator.h, format.h) will lead to unity_test failure?! Checking ...